src/status_im/ui/screens/wallet/events.cljs

```diff
@@ -124,7 +124,7 @@
   :update-transactions-success
   (fn [db [_ transactions]]
     (-> db
-        (assoc-in [:wallet :transactions] transactions)
+        (update-in [:wallet :transactions] #(merge % transactions))
```

Comment:
Could be just `(update-in [:wallet :transactions] merge transactions)`

Reply:
If you do this then the transactions that are not confirmed will never be updated. `%` is the current transaction history (including the one that was derived from the unsigned transaction and is not yet confirmed in the chain) so you want its values to be overriden by `transactions` which is data from etherscan

Reply:
@yenda i'm just saying that 
`(update-in [:wallet :transactions] merge transactions)` is equal to
`(update-in [:wallet :transactions] #(merge % transactions))`
but shorter, no need to create anonymous fn

Reply:
yes you are correct